packages/theming/apple-theme/src/appleTheme.colors.ios.ts

```diff
@@ -0,0 +1,223 @@
+import { ThemeColorDefinition } from '@fluentui-react-native/theme-types';
+import { ApplePalette } from './fluentAppleColors.ios';
+import { AppleSemanticPalette } from './appleSemanticColors.ios';
+
+/** creates a palette of colors for the apple theme, given the FluentUI Apple Palette and Apple Semantic Palette */
+export function paletteFromAppleColors(fluentApple: ApplePalette, appleSemantic: AppleSemanticPalette): ThemeColorDefinition {
+  return {
+    /* PaletteBackgroundColors & PaletteTextColors */
+
+    background: appleSemantic.systemBackground,
+    bodyStandoutBackground: appleSemantic.secondarySystemBackground,
+    bodyFrameBackground: appleSemantic.systemBackground,
+    bodyFrameDivider: appleSemantic.opaqueSeparator,
+    bodyText: appleSemantic.label,
+    bodyTextChecked: appleSemantic.label,
+    subText: appleSemantic.secondaryLabel,
+    bodyDivider: appleSemantic.separator,
+
+    // TODO
+    disabledBackground: fluentApple.gray100,
+    disabledText: appleSemantic.tertiaryLabel,
+    disabledBodyText: appleSemantic.quaternaryLabel,
+    disabledSubtext: appleSemantic.tertiaryLabel,
+    disabledBodySubtext: appleSemantic.quaternaryLabel,
+
+    focusBorder: 'transparent',
+    variantBorder: appleSemantic.separator,
+    variantBorderHovered: appleSemantic.separator,
+    defaultStateBackground: appleSemantic.systemFill,
+
+    // TODO
```

Comment:
Add documentation to the TODOs in this file?

Reply:
I should just remove all of these. They were there while I was still mapping all the colors but now that I have a complete set, I can probably just remove them, and let people comment on the mappings